SimTKcommon/Geometry/include/SimTKcommon/internal/DecorativeGeometry.h

```diff
@@ -596,6 +596,79 @@ class SimTK_SimTKCOMMON_EXPORT DecorativeMeshFile : public DecorativeGeometry {
 };
 
 
+/** This defines a displayable torus  . **/
+class SimTK_SimTKCOMMON_EXPORT DecorativeTorus : public DecorativeGeometry {
+public:
+    explicit DecorativeTorus(Real inner=0.1, Real outer=0.5);
```

Comment:
For consistency, this Torus definition should match Simbody's existing `ContactGeometry::Torus` object that Ian wrote ([click here to see definition](https://github.com/simbody/simbody/blob/master/SimTKmath/Geometry/include/simmath/internal/ContactGeometry.h#L1377)). That specifies how the torus is situated in its local frame, and gives the geometry by the centerline radius and the "tube radius" (if the tube radius is zero it degenerates into a circle).

There should also be a longer comment here describing exactly what you get with this Decoration. Probably the `ContactGeometry::Torus` comment will work.